Automattic/vip-go-mu-plugins
# Updating Parsely plugin loading hook

## Description

The release of Parse.ly 2.6 comes with some changes on how the code is structured. One of the main differences is that the plugin now hooks to `plugins_loaded` to initialize itself. We are currently loading `wp-parsely` on mu-plugins by using the `after_setup_theme` hook. That hook occurs after `plugins_loaded`, hence, this last one would never be triggered and the Parse.ly plugin would never be initialized.

This PR changes how the plugin is loaded by using a hook that fires earlier, `muplugins_loaded`.

## Changelog Description

### Updating Parsely plugin loading hook

Changing how Parse.ly plugin is loaded to accommodate wp-parsely 2.6.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Load Parse.ly 2.6 (everything should work normally).
3. Check out master and try to load Parse.ly. Site should fatal.
